qualtran/bloqs/mcmt/multi_control_multi_target_pauli.py

```diff
@@ -111,43 +101,61 @@ class MultiControlPauli(GateWithRegisters):
     target_gate: cirq.Pauli = cirq.X
```

Comment:
does it make sense to have a default here? do you have to use cirq gates or can you use `XGate()`?

Reply:
Happy to remove the default in favour of more readability. 

Right now, it needs to be a cirq gate because `Bloq.controlled` and `Gate.controlled` don't alight very well and `Gate.controlled` is nicer for `X/Y/Z` gates because it has special overrides to give back `CX/CY/CZ` and `CCX/CCY/CCZ` for `1/2` qubit case. In future, I want to accept both `XGate` and `cirq.X` here

Reply:
@mpharrigan I've removed the default, PTAL!